login/login-page.xml

```diff
@@ -3,16 +3,47 @@
     <ActionBar title="Login" icon="" class="action-bar">
     </ActionBar>
 
-     <StackLayout>
-        <Button text="Log in with Facebook" tap="onLoginFacebookButtonTap"></Button>
-        <Button text="Log in with Google +" tap="onLoginGoogleButtonTap"></Button>
-        <Label text="or"></Label>
+     <StackLayout class="p-t-15">
+        <Button class="btn btn-outline text-left"  
+            tap="onLoginFacebookButtonTap">
+            <FormattedString>
```

Comment:
We should probably follow the [docs](https://docs.nativescript.org/cookbook/formatted-string-ng)

Reply:
I have used the same syntax as in master detail templates  - [here](https://github.com/NativeScript/template-master-detail/blob/master/cars/cars-list-page.xml) and [here](https://github.com/NativeScript/template-master-detail-kinvey/blob/master/cars/cars-list-page.xml), in order to be consistent :)